addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1905,6 +1904,37 @@ options.registry.collapse = options.Class.extend({
     },
 });
 
+options.registry.Accordion = options.Class.extend({
+    /**
+     * Toggle the position of the cards' collapse state icons
+     * (either floats left (default) or right)
+     *
+     * @see this.selectClass for parameters
+     */
+    iconPos: function (previewMode, widgetValue, params) {
+        const $cards = this.$('.accordion .card');
```

Comment:
You could find the card headers directly instead of searching for the cards then the card headers inside.